src/components/panel/panel.spec.js

```diff
@@ -1450,6 +1633,41 @@ describe('$mdPanel', function() {
     attachedElements.push(element);
   }
 
+  function clickOutsideToClosePanel() {
```

Comment:
nit: This function performs a click. It doesn't close the panel. Rename this to something like: 'clickPanelContainer`